src/features/CourtCaseDetails/Tabs/Panels/DefendantDetails.tsx

```diff
@@ -13,38 +13,98 @@ import ErrorPromptMessage from "components/ErrorPromptMessage"
 import { BailConditions } from "./BailConditions"
 import { createUseStyles } from "react-jss"
 import { useCourtCase } from "../../../../context/CourtCaseContext"
+import React, { useState } from "react"
+import { isAsnFormatValid } from "@moj-bichard7-developers/bichard7-next-core/core/phase1/lib/isAsnValid"
+import { AmendmentKeys, AmendmentRecords, IndividualAmendmentValues } from "../../../../types/Amendments"
+import EditableFieldTableRow from "../../../../components/EditableFieldTableRow"
+import Phase from "@moj-bichard7-developers/bichard7-next-core/core/types/Phase"
+
+interface DefendantDetailsProps {
+  amendmentRecords: AmendmentRecords
+  amendFn: (keyToAmend: AmendmentKeys) => (newValue: IndividualAmendmentValues) => void
+}
 
 const useStyles = createUseStyles({
   wrapper: {
     "& td": {
       width: "50%"
     }
+  },
+
+  asnInput: {
+    width: "15rem"
   }
 })
 
-export const DefendantDetails = () => {
+export const DefendantDetails = ({ amendFn, amendmentRecords }: DefendantDetailsProps) => {
   const classes = useStyles()
   const courtCase = useCourtCase()
   const defendant = courtCase.aho.AnnotatedHearingOutcome.HearingOutcome.Case.HearingDefendant
-  const asnErrorPrompt = findExceptions(
+  const asnSystemErrorExceptionPrompt = findExceptions(
     courtCase,
     courtCase.aho.Exceptions,
     ExceptionCode.HO200113,
     ExceptionCode.HO200114
   )
+  const updatedAsn =
+    courtCase.updatedHearingOutcome?.AnnotatedHearingOutcome?.HearingOutcome?.Case?.HearingDefendant
+      ?.ArrestSummonsNumber
+  const [isValidAsn, setIsValidAsn] = useState<boolean>(true)
+  const handleAsnChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    const asn = e.target.value.toUpperCase()
+    setIsValidAsn(isAsnFormatValid(asn))
+    amendFn("asn")(asn)
+  }
+
+  const asnFormGroupError = isValidAsn ? "" : "govuk-form-group--error"
+  const isAsnEditable =
+    courtCase.canUserEditExceptions && courtCase.aho.Exceptions.length > 0 && courtCase.phase === Phase.HEARING_OUTCOME
```

Comment:
`courtCase.aho.Exceptions.length > 0`

 don't we need to check if it is an ASN exception?

Reply:
No. As long as there is an unresolved exception, we want to make it editable.

Reply:
Then I think its redundant, `.canUserEditExceptions` already checks if error status is unresolved(should only be unresolved if there are unresolved exceptions) 
```
  const isAsnEditable =
    courtCase.canUserEditExceptions && courtCase.phase === Phase.HEARING_OUTCOME
```

Reply:
`courtCase.aho.Exceptions.length > 0`  is removed and updated an e2e test